script/bootstrap

```diff
@@ -66,7 +66,7 @@ function bootstrap() {
 
   var buildInstallCommand = initialNpmCommand + npmFlags + 'install';
   var buildInstallOptions = {cwd: path.resolve(__dirname, '..', 'build')};
-  var apmInstallCommand = npmPath + npmFlags + '--target=0.10.35 ' + 'install';
+  var apmInstallCommand = npmPath + npmFlags + '--target=4.2.1 ' + 'install';
```

Comment:
Have you tested by removing the `target` argument? This is a code smell. We should not be be hardcoding versions in build scripts in order to avoid the "nightmare" scenario we just experienced.